docs/marks.rst

```diff
@@ -113,42 +113,101 @@ You can then run tests that are relevant only to the selected ticket:
     cd src/tests/system
     pytest --mh-config=mhc.yaml --mh-lazy-ssh -v --ticket=gh#3433
 
-@pytest.mark.tier
-=================
+@pytest.mark.importance
+=======================
 
-The `tier mark <https://github.com/next-actions/pytest-tier>`__ can
-associate a test with a specific tier.
+The `importance mark <https://github.com/next-actions/pytest-importance>`__ can
+associate a test with a level of importance. This is used by quality engineers to
+prioritize and group the test determined by the level of impact to the customer.
 
-The ``@pytest.mark.tier`` takes single number as an argument.
+The ``@pytest.mark.importance`` takes a string as an argument. The values used
+are "critical", "high", "medium" and "low".
 
 .. code-block:: python
     :caption: Examples
 
-    @pytest.mark.tier(0)
-    def test_tier0()
+    @pytest.mark.importance("critical")
+    def test_importance_critical()
         pass
 
-    @pytest.mark.tier(1)
-    def test_tier1()
+    @pytest.mark.importance("high")
+    def test_importance_high()
         pass
 
-You can then run tests that are relevant only to the selected ticket:
+You can then run the tests by importance:
 
 .. code-block:: text
 
     cd src/tests/system
-    pytest --mh-config=mhc.yaml --mh-lazy-ssh -v --tier=1
+    pytest --mh-config=mhc.yaml --mh-lazy-ssh -v --importance="critical"
 
-Tier definition
-===============
+Importance definition
+=====================
 
-.. list-table:: Tier definition
-    :align: center
+.. list-table::
```

Comment:
I would avoid using table and just do it in bullet points. The description is long and it does not show up nicely in the table since it does not support multiline fields.

```rst
* **critical**: description ...
```

Same for the custom markers.